app/views/web/404.html.haml

```diff
@@ -0,0 +1,4 @@
+%h1
+  Ooops! La página que buscas no existe
```

Comment:
If you're creating translations for all the fields, perhaps its also fine to create for all the rest of texts in the website, to avoid future lack of translations...